openshift/template.yaml

```diff
@@ -144,6 +144,25 @@ parameters:
 - name: LIVENESS_PROBE_INITIAL_DELAY_SECONDS
   value: "30"
   required: false
+- name: ENVOY_IMAGE
+  value: "docker.io/envoyproxy/envoy"
+  required: false
+- name: ENVOY_TAG
+  value: "v1.24.0"
+  required: false
+- name: ENVOY_IMAGE_PULL_POLICY
+  value: IfNotPresent
+  required: false
+- name: ENVOY_MEMORY_REQUEST
+  value: 256Mi
+- name: ENVOY_CPU_REQUEST
+  value: 10m
+- name: ENVOY_MEMORY_LIMIT
+  value: 512Mi
+- name: ENVOY_PORT
+  value: 8900
+- name: SVC_TARGET_PORT
+  value: 8900
```

Comment:
I don't see the need to make the ports configurable. If you decide to do so then use the variables instead of the constants everywhere below.

Reply:
:+1: makes sense, however I'd like to keep the svc target configurable, in case something goes wrong when deploying envoy we can easily switch to the original port and bypass envoy by overriding this parameter (no need to backport changes in the git repo). WDYT?

Reply:
That is a good reason, yes.